### PR TITLE
[Website]: Fix line breaks on typography page

### PR DESCRIPTION
**Pull request by maya**
## Description

Fixes line breaks on typography page for body text and italic text.

## Additional information

This is what it looks like on mobile:

screen shot 2016-05-31 at 3 41 53 pm

This is what it looks like on desktop:

screen shot 2016-05-31 at 3 42 07 pm

Fixes: #615.

### CODE REVIEW

**Comment by juliaelman** [comment]
@maya is there a reason why a new class name was added instead of incorporating this change into this line? https://github.com/18F/web-design-standards/blob/017fe1d45461b90b7b1322e532e4fde8628eeadf/docs/doc_assets/css/styleguide.scss#L817

**Comment by maya** [comment]
@juliaelman it's a separate rule for just those longer text examples. That line groups heading elements (h1-h6) which it does not apply to.

File: docs/_visual/typography.md [hunk 1]
```diff
@@ -705,7 +705,7 @@ order: 01
             </p>
           </div>
           <div class="usa-font-example usa-width-one-half">
-            <p><em>Italic body copy. A series of <br>sentences together which <br>make a paragraph.</em></p>
+            <p class="font-example-paragraph"><em>Italic body copy. A series of <br>sentences together which <br>make a paragraph.</em></p>
```
Comment (juliaelman) [inline docs/_visual/typography.md]: Is there a reason why the line breaks were left in here?

Comment (maya) [inline docs/_visual/typography.md]: Removed that and the remaining `<br>`s in 2d81e6c.

**Comment by juliaelman** [comment]
LGTM - thanks Maya!

**Pull request merged**